resources/nl-core/ext-CopyIndicator.xml

```diff
@@ -40,7 +45,7 @@
     <expression value="Device" />
   </context>
   <type value="Extension" />
-  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension"/>
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />

```

Comment:
@pieter-edelman-nictiz is deze spatie nodig?

Reply:
The check is not (cannot be) very smart. It's just: "when a short is defined, a mapping must also be defined".

This PR is not about missing mappings but about fixing validation errors. Please open a new PR when you want to ad them.

Reply:
Spatie is niet nodig en kan geen kwaad. De ene editor doet iets anders dan de andere.